client/src/pages/MyJobs/MyJobs.tsx

```diff
@@ -0,0 +1,20 @@
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+
+const MyJobs = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Grid className={classes.myJobs}>
```

Comment:
The MUI `Box` component has flexbox built into it, you can use that here instead.

Eg: `<Box display="flex" alignItems="center"> </Box>`

Reply:
Applied